extensions/hashicorp-vault/src/test/resources/logback.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like this file was inadvertently added

Reply:
Yes, this was needed downstream. Removed it.